### PR TITLE
Rapdy: Add idempotency for all endpoints

### PR DESCRIPTION
**Pull request by almalee24**

### CODE REVIEW

File: lib/active_merchant/billing/gateways/rapyd.rb [hunk 1]
```diff
@@ -74,10 +77,12 @@ def store(payment, options = {})
         add_payment_fields(post, options)
         add_payment_urls(post, options, 'store')
         add_address(post, payment, options)
+        add_idempotency(options)
         commit(:post, 'customers', post)
       end
 
       def unstore(customer)
+        add_idempotency(options)
```
Comment (bradbroge) [inline lib/active_merchant/billing/gateways/rapyd.rb:85]: I don't think `options` exists in this context.

**Review by bradbroge — COMMENTED** [review]
Question around the `unstore` action

File: lib/active_merchant/billing/gateways/rapyd.rb (hunk 1 above)
Comment (almalee24) [reply lib/active_merchant/billing/gateways/rapyd.rb:85]: Right, why didn't the unit tests catch that. Let me look at that.

**Comment by aenand** [comment]
Why would someone want idempotency for all endpoints?

File: lib/active_merchant/billing/gateways/rapyd.rb (hunk 1 above)
Comment (almalee24) [reply lib/active_merchant/billing/gateways/rapyd.rb]: Fix it

**Comment by almalee24** [comment]
> Why would someone want idempotency for all endpoints?

I'm not sure why they would want to but if they did pass it all these endpoints support it.

**Review by bradbroge — APPROVED** [review]
👍🏻

**Pull request merged**